src/setupTest.js

```diff
@@ -39,6 +39,10 @@ mergeConfig({
   LEARNING_BASE_URL: process.env.LEARNING_BASE_URL,
   EXAMS_BASE_URL: process.env.EXAMS_BASE_URL || null,
   CALCULATOR_HELP_URL: process.env.CALCULATOR_HELP_URL || null,
+  ACCOUNT_PROFILE_URL: process.env.ACCOUNT_PROFILE_URL || null,
+  ACCOUNT_SETTINGS_URL: process.env.ACCOUNT_SETTINGS_URL || null,
+  IGNORED_ERROR_REGEX: process.env.IGNORED_ERROR_REGEX || null,
+  MFE_CONFIG_API_URL: process.env.MFE_CONFIG_API_URL || null,
```

Comment:
I'm not sure why these lines are included in this PR? Are they required for some of the new tests?